### ISSUE TITLE
that was stupid

### ISSUE THREAD

**Issue by LouisGac** [QiushiWu/qiushiwu.github.io#12]
and you know it.

**Comment by ABotond** [comment]
It is also stupid opening the next unproductive issue with just bashing the author.
Does this make feel you better?
Does this make the situation any better?
Did you at least take the time and read the paper?

**Comment by LouisGac** [comment]
> Does this make feel you better?

yes. I feel like a super hero fighting agains a super vilain. You can call me the Internet Batman

> Does this make the situation any better?

the situation is not that bad. But Qiushi should learn about Kant's kategorischer Imperativ. If everybody would act like him, how would be the situation? So public shaming is just a way to avoid that. 

> Did you at least take the time and read the paper?

I'm having a look, and I've just followed the author. So just calm down.

**Comment by XiaoxingChen** [comment]
Those who are cowards in real world tend to retrieve their fragile confidence on Internet and then says: feel himself like a `super hero`.

**Comment by LouisGac** [comment]
> The cowards in real life tend to retrieve their fragile confidence on the virtual Internet and then say:
> 
> > I feel like a super hero

ah, here the super vilain.

so I tell you without fear, without fear, I dare to denounce:  booooooh

**Comment by ABotond** [comment]
kindergarden...

**Comment by LouisGac** [comment]
well at least I didn't upset the whole Linux community with my childishness  ^^

**Comment by loneicewolf** [comment]
> and you know it.

I think this is quite understandable.. I think it's quite obvious, However - I do agree pointing out specifically what  - is what *I advise doing * but still, it's pretty obvious.

Have a good day on everyone!

//Will.
